.editorconfig

```diff
@@ -0,0 +1,31 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+trim_trailing_whitespace = true
+max_line_length = 120
+ij_visual_guides = 100,120,150
```

Comment:
This one is specific to intelligy ? A comment would be appreciated

Reply:
Yes, this applies to IntelliJ editors - it adds visual text borders. Doesn't affect the code in any way, but it's easier to understand that you're approaching the right border of the text width

![image](https://github.com/user-attachments/assets/cbb49c22-f41a-4026-a3ca-f0512f397a89)

Reply:
So yes, a comment would help

Something like

# visual setting for intelligy IDE, no consequences on code validation